extensions/amp-imgur/0.1/amp-imgur.js

```diff
@@ -79,10 +79,10 @@ export class AmpImgur extends AMP.BaseElement {
     iframe.setAttribute('frameborder', '0');
     iframe.setAttribute('allowfullscreen', 'true');
 
-    iframe.src =
-      'https://imgur.com/' +
-      encodeURIComponent(this.imgurid_) +
-      '/embed?pub=true';
+    const sanitizedID = this.imgurid_.startsWith('a/')
```

Comment:
Is `a/` the only pattern change? Can the values also include `b/` etc?

Reply:
offline conversation: Seems like `a/` stands for an album/gallery although there is nothing about it on the `imgur` docs. We'll keep it this way for now until another format surfaces.